common/changes/iconame-reducesize_2017-01-10-01-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Icon: -BREAKING CHANGE- Change IconName from an enum to type to greatly reduce file size. Any component that is currently consuming the icon enum will break.",
```

Comment:
We're doing this because it adds 7k gzipped to the download size which is gigantic! It's a no brainer. However if you're asking why minor bump instead of major... I've always assumed semantically that 1.0.0 is not "pre" release.